README.md

```diff
@@ -1220,6 +1220,20 @@ Vagrant.configure(2) do |config|
 end
 ```
 
+## Custom command line arguments
+You can also specify multiple qemuargs arguments for qemu-system
+
+* `value` - Value
+
+```ruby
+Vagrant.configure("2") do |config|
+  config.vm.provider :libvirt do |libvirt|
+    libvirt.qemuargs :value => "-device"
+    libvirt.qemuargs :value => "isa-applesmc,osk=ourhardworkbythesewordsguardedpleasedontsteal(c)AppleComputerInc"
```

Comment:
Please do not put this example string in the documentation; I'm sure you can find other examples to put there.